collector-bitmovin-player/src/test/java/com/bitmovin/analytics/analytics/BitmovinUtilTest.java

```diff
@@ -0,0 +1,45 @@
+package com.bitmovin.analytics.analytics;
+
+import com.bitmovin.analytics.CollectorConfig;
+import com.bitmovin.analytics.bitmovin.player.BitmovinUtil;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class BitmovinUtilTest {
+    @Test
+    public void testGetIsLiveFromConfigOrPlayer_ReturnsConfigValueTrueIfPlayerNotReady() throws Exception {
+        CollectorConfig config = new CollectorConfig();
+        config.setIsLive(true);
+
+        boolean isLive = BitmovinUtil.getIsLiveFromConfigOrPlayer(false, config, false);
+        assertTrue(isLive);
```

Comment:
```suggestion
        assertThat(isLive).isTrue();
```
aren't we using the assertJ assertions now?